src/main/java/net/poweredbyhate/wildtp/WildTP.java

```diff
@@ -12,54 +12,42 @@
 import org.bukkit.plugin.RegisteredServiceProvider;
 import org.bukkit.plugin.java.JavaPlugin;
 import org.bukkit.scheduler.BukkitRunnable;
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.UUID;
-import java.util.concurrent.Callable;
+import me.ryanhamshire.GriefPrevention.DataStore;
+import me.ryanhamshire.GriefPrevention.GriefPrevention;
+import net.milkbowl.vault.economy.Economy;
 
 /**
  * Created by John on 5/5/2016.
  */
 public class WildTP extends JavaPlugin {
-
     private boolean wamuppahTooCool = false;
-
-    static boolean isDebug;
-
-    public static WildTP instace;
-    public static TooWildForEnums enums = new TooWildForEnums();
-    public static PortalzGoneWild portalz = new PortalzGoneWild();
-
-    public static Economy econ;
-    public static ConfigurationSection randomeWorlds;
-    public static boolean useRandomeWorldz;
-    public static boolean newPlayersTeleported;
-    public static boolean useExperimentalChekar;
-    public static boolean useOtherChekar;
-    public static boolean noCreditJustCash;
-    public static DataStore dataaaastorege;
-    public static boolean wb;
-    public static boolean notPaper;
-    public static Location cash;
-    String[] bluredLines;
-    HashMap<String, String> aliaz;
-    HashMap<String, WorldConfig> thugz;
+    // @formatter:off
+    static boolean
+        enableUselessGUI, isDebug, newPlayersTeleported, noCreditJustCash, notPaper,
+        useExperimentalChekar, useOtherChekar, useRandomeWorldz, wb;
+    // @formatter:on
+    static ConfigurationSection   randomeWorlds;
+    static DataStore              dataaaastorege;
+    static Economy                econ;
+    static Location               cash;
+    static TooWildForEnums        enums;
+    static WildTP                 instace;
+    String[]                      bluredLines;
+    PortalzGoneWild               portalz;
+    HashMap<String, String>       aliaz;
+    HashMap<String, WorldConfig>  thugz;
     HashMap<UUID, BukkitRunnable> ohWait;
 
     public void onEnable() {
         if (Integer.valueOf(Bukkit.getBukkitVersion().split("\\.")[1]) <= 12) {
             getLogger().severe("This version of Wilderness-TP does not support your ancient server version.");
             getLogger().warning("Either update your server to 1.13, or use Wild 1.52");
-            getLogger().warning("http://r.robomwm.com/oldwild");
+            getLogger().warning("https://robomwm.com"); // URL changed because the old one no more works

```

Comment:
So somehow this commit reverted this commit https://github.com/AcmeProject/WildernessTp/pull/44/commits/dcf96b7decd1c1f0ffd5b7b406ecc6eb20dbb054

I'm guessing you might've not pulled changes locally and just copy-pasted your local repo over the remote repo to push this in.